vector/src/main/java/im/vector/app/features/createdirect/CreateDirectRoomViewModel.kt

```diff
@@ -146,24 +155,7 @@ class CreateDirectRoomViewModel @AssistedInject constructor(
                     {
                         setState { copy(isLoading = false) }
                         _viewEvents.post(CreateDirectRoomViewEvents.Failure(it))
-=======
-            val adminE2EByDefault = rawService.getElementWellknown(session.sessionParams)
-                    ?.isE2EByDefault()
-                    ?: true
-
-            val roomParams = CreateRoomParams()
-                    .apply {
-                        selections.forEach {
-                            when (it) {
-                                is PendingSelection.UserPendingSelection     -> invitedUserIds.add(it.user.userId)
-                                is PendingSelection.ThreePidPendingSelection -> invite3pids.add(it.threePid)
-                            }
-                        }
-                        setDirectMessage()
-                        enableEncryptionIfInvitedUsersSupportIt = adminE2EByDefault
```

Comment:
What is new among this part in EA ?

Reply:
We have modified a lot this class in Tchap, I probably made the choice to remove the unused code for clarity

Reply:
@Claire1817 could you take a look on file history to verify that's florian said.